src/Sulu/Bundle/ContentBundle/Command/PHPCRInitCommand.php

```diff
@@ -45,6 +45,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $workspace = $session->getWorkspace();
 
         // init node namespace and types
+        // TODO move this to initializer
```

Comment:
Probably already solved in https://github.com/sulu-io/sulu/pull/2076 (can you check?). If this is the case I would already remove that comment.